source/extensions/filters/listener/tls_inspector/tls_inspector.h

```diff
@@ -23,7 +23,6 @@ namespace TlsInspector {
   COUNTER(connection_closed)                                                                       \
   COUNTER(client_hello_too_large)                                                                  \
   COUNTER(read_error)                                                                              \
-  COUNTER(read_timeout)                                                                            \
```

Comment:
Looks like there aren't docs to update here.  Non-blocking but maybe file a bug against Greg or Piotr to doc up the stats we have?

Reply:
https://github.com/envoyproxy/envoy/issues/5235